app/assets/stylesheets/sephora_style_guide/partials/product-card.scss

```diff
@@ -0,0 +1,172 @@
+.product-card {
+  width: 100%;
+
+  .product-card-image {
+    width: 100%;
+    display: block;
+  }
+
+  .product-card-actions {
+    position: relative;
+    width: 100%;
+    height: $product-card-actions-height;
+
+    .product-card-add-to-bag,
+    .product-card-add-to-wish-list {
+      margin-top: -$product-card-actions-height;
+
+      @include media-breakpoint-down(lg) {
+        display: none;
+      }
+    }
+  }
+
+  &:hover {
+    .product-card-wish-list-heart {
+      display: block;
+    }
+  }
+
+  .out-of-stock {
+    @include font-avalon-demi;
+    top: 37%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 90%;
+    height: $product-card-actions-height;
+    background-color: rgba(255, 255, 255, 0.75);
+    position: absolute;
```

Comment:
While I don't do the "sort rules by alphabetical order" thing, I like to see rules grouped by the common intent they have. For example, the ones that are used for flexbox should be one after another; position could be `position`, `top`, `right`, `height`, `width`; text related could be `font-size`, `color`...